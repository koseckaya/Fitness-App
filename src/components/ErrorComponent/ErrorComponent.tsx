
import './ErrorComponent.scss';
import { Container } from '../Container';

export type Props = {
    className?: string;
};

export const ErrorComponent = () => {
 
    
    return (
        <Container>
            <div className='error-component' >
                <div className='error-title'>Something went wrong</div>
                <div className='error-message'>Let's try later</div>
            </div>
        </Container>
    )
};


