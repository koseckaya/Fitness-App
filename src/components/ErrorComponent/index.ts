export { ErrorComponent } from './ErrorComponent';
export type { Props as ErrorProps } from './ErrorComponent';
