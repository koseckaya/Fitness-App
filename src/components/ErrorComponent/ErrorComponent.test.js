import React from 'react';
import renderer from 'react-test-renderer';
import { ErrorComponent } from './ErrorComponent';


it('renders correctly', () => {
    const tree = renderer
        .create(<ErrorComponent></ErrorComponent>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});