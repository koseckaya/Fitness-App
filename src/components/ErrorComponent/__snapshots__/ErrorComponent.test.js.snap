// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container "
>
  <div
    className="error-component"
  >
    <div
      className="error-title"
    >
      Something went wrong
    </div>
    <div
      className="error-message"
    >
      Let's try later
    </div>
  </div>
</div>
`;
